lib/path/default.nix

```
@@ -336,6 +336,37 @@ in /* No rec! Add dependencies on this file at the top. */ {
               ${subpathInvalidReason path}''
       ) 0 subpaths;
 
+  /*
+  Split [a subpath](#function-library-lib.path.subpath.isValid) into its path component strings.
+  Throw an error if the subpath isn't valid.
+  Note that the returned path components are also valid subpath strings, though they are not [normalised](#function-library-lib.path.subpath.normalise).
```

```suggestion
  Note that the returned path components are also valid subpath strings, though they are intentionally not [normalised](#function-library-lib.path.subpath.normalise).
```

Reinforces that path component strings aren't supposed to be normalized, having `./`. In case someone thinks this function can be improved.